pkg/deploy-manager/storagecluster.go

```diff
@@ -181,6 +181,21 @@ func (t *DeployManager) createStorageCluster() (*ocsv1.StorageCluster, error) {
 	return newSc, nil
 }
 
+// deleteStorageCluster is used to delete the test suite storage cluster
+func (t *DeployManager) deleteStorageCluster() error {
```

Comment:
Is deleting the StorageCluster enough here? I think we should take care of the CephCluster finalizer so that deletion is faster.

Reply:
hmm, when I tested it on my local cluster the tests were passing without the removal of finalizers. But removing the finalizer will be a good idea. Added that.